tests/storage/storageserver_test.py

```diff
@@ -250,6 +250,29 @@ def test_glusterfs_cli_missing(self, monkeypatch):
         gluster = GlusterFSConnection(id="id", spec="192.168.122.1:/music")
         assert gluster.options == ""
 
+    def test_glusterfs_backup_serv_connection(self, monkeypatch):
+        monkeypatch.setattr(storageServer, 'supervdsm', FakeSupervdsm())
+        monkeypatch.setattr(gluster_cli, 'exists', lambda: True)
+
+        def glusterVolumeInfo(volname=None, volfileServer=None):
+            assert volname == "music"
+            if volfileServer is not None:
+                raise ge.GlusterException()
+            return {'music': {'brickCount': '2',
+                              'bricks': ['192.168.122.1:/tmp/music',
+                                         '192.168.122.2:/tmp/music']
+                              }
+                    }
+        storageServer.supervdsm.glusterVolumeInfo = glusterVolumeInfo
+        user_options = "backup-volfile-servers=192.168.122.1:192.168.122.2"
+        gluster = GlusterFSConnection(id="id", spec="192.168.122.3:/music",
+                                      options=user_options)
+        expected_volinfo = {'brickCount': '2',
+                            'bricks': ['192.168.122.1:/tmp/music',
+                                       '192.168.122.2:/tmp/music']
+                            }
+        assert gluster.volinfo == expected_volinfo
```

Comment:
The test looks right for the current code, but need to change if the code will always retry.